src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -256,7 +256,9 @@ export class ActivityGraphFillComputer {
     if (enableCPUUsage && threadCPUDelta && threadCPUDelta[lastIdx] !== null) {
       const cpuDelta = threadCPUDelta[lastIdx];
       const intervalDistribution =
-        (samples.time[lastIdx] - samples.time[lastIdx - 1]) / interval;
+        lastIdx === 0
+          ? 1
+          : (samples.time[lastIdx] - samples.time[lastIdx - 1]) / interval;
```

Comment:
There is actually a similar code above in the line 228. I had removed the `idx === 0` case before, thinking that it would come to this code only at the last sample. But when there is only one sample in the view, the first sample is also the last sample 😄